root/programs/C#/Frameworks/Infrastructure/Public/Util/GetPassword.cs

```diff
@@ -27,24 +27,58 @@
 //*
 //*  日時        更新者            内容
 //*  ----------  ----------------  -------------------------------------------------
-//*  2017/01/10  西野  大介        新規作成
+//*  2017/01/10  西野 大介         新規作成
 //**********************************************************************************
 
-// System
 using System;
+using System.Collections.Generic;
 using System.Security.Cryptography;
 
+using Touryo.Infrastructure.Public.Str;
+
 namespace Touryo.Infrastructure.Public.Util
 {
     /// <summary>GetPassword</summary>
     public class GetPassword
     {
-        /// <summary>Generate</summary>
-        /// <param name="length">長さ</param>
-        /// <param name="numberOfNonAlphanumericCharacters">Alphanumeric以外の文字数</param>
+        /// <summary>Base64Secretを生成</summary>
+        /// <param name="keySize">int</param>
```

Comment:
@yusukemaegawa 

I see.
There is a possibility that similar problems may exist elsewhere,
but we will modify only this for this time.